packages/back-end/src/app/controllers/easy-genomics/organization/user/request-organization-user.lambda.ts

```diff
@@ -12,6 +13,9 @@ export const handler: Handler = async (
 ): Promise<APIGatewayProxyResult> => {
   console.log('EVENT: \n' + JSON.stringify(event, null, 2));
   try {
+    if (!validateSystemAdminAccess(event)) {
```

Comment:
I think this API should also be accessible to OrganizationAdmins too, so the IF statement should be:

```
    if (!validateSystemAdminAccess(event) && !validateOrganizationAdminAccess(event, request.OrganizationId)) {
      throw new UnauthorizedAccessError();
    }
```